assign.py

```diff
@@ -256,6 +263,10 @@ def main():
             era = options.era
         elif not taskchain:
             era = wf.info['AcquisitionEra']
+        #Set era and procstring to none for merge ACDCs inside a task chain
+        if schema["RequestType"] == "Resubmission") and wfInfo.info["PrepID"].startswith("task") and "Merge" in schema["InitialTaskPath"].split("/")[-1]:
```

Comment:
Despite of the SyntaxError, I am getting this error "NameError: global name 'wfInfo' is not defined", "wfInfo" does not exits,  the variable is "wf".